src/style.css

```diff
@@ -1,3 +1,89 @@
-body {
+/* #region ZERO */
+body{
+  padding: 0;
   margin: 0;
+  font-family: Roboto, sans-serif;
+  padding-bottom: 2000px;
+}
+
+.wrapper{
+  max-width: 1200px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+:root{
+  --header-height: 60px;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+/* #endregion ZERO */
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  object-fit: contain;
+  padding: 10px 0;
+}
+
+nav{
+  text-transform: uppercase;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav_item{
```

Comment:
The class name `.nav_item` should be `.nav__item` to maintain consistency with BEM naming conventions used elsewhere in your CSS.